userapi/src/main/java/fk/prof/userapi/api/AggregatedProfileLoader.java

```diff
@@ -28,6 +32,12 @@
  */
 public class AggregatedProfileLoader {
 
+    private static String profileLoadTimerPrefix = "profile.load.timer";
+    private static String profileSummaryLoadTimerPrefix = "profilesummary.load.timer";
+
+    private Meter profileLoadFailureCounter = Util.meter("profile.load.failures.meter");
```

Comment:
Can we move these metrics to `fk.prof.metrics.MetricName` like  enum? Improves readability, allows renaming/changing metric names efficiently and more importantly, that is the only place I need to check to find out all the metrics being exposed and which one will help me in debugging a certain problem
You can consider renaming `MetricName` to `BackendMetricNames` and the new one to `UserapiMetricNames`